src/fortran_routing/Reservoir_Binding/kernels/setup.py

```diff
@@ -0,0 +1,38 @@
+from distutils.core import setup
+from distutils.extension import Extension
+from Cython.Distutils import build_ext
+
+
+#Document to set this up
+#When installing this module from source
+#$LD_LIBRARY_PATH=/path/to/my/netcdf/:$LD_LIBRARY_PATH setup.py
+
```

Comment:
TODO: Imitate more general usage demonstrated in `src/python_routing_v02/setup.py` (__OR__ integrate directly into that file...)

Reply:
I added this todo in the code. As we integrate the reservoirs, we can decide to integrate these extensions into the main setup.py, or I will refactor this reservoir setup.py to match that format.